webgallery/models.py

```diff
@@ -19,23 +19,23 @@ def __str__ (self):
       return "%s" % (self.title)
 
    
-class Slider (models.Model):
-   title = models.CharField(max_length = 200)
-   picture = models.ManyToManyField(Picture)
-   
-   def __str__ (self):
-      return "%s" % (self.title)
-   
-   def get_pictures(self):
-      return self.picture.all()
+#class ActivePic (models.Model):
+#   title = models.CharField(max_length = 200)
+#   picture = models.ManyToManyField(Picture)
+#   
+#   def __str__ (self):
+#      return "%s" % (self.title)
+#   
+#   def get_pictures(self):
+#      return self.picture.all()
 
 
-class Project (models.Model):
+class Gallery (models.Model):
    title = models.CharField(max_length = 200)
    date = models.DateTimeField(auto_now_add = True)
```

Comment:
What date is it? non descriptive name.
Is it creation date? Last modification date?